### DISCUSSION TITLE
Ignition is adding a hash to my urls?

### DISCUSSION THREAD

**Discussion by nickdekruijk** [spatie/laravel-ignition#54] [General]
Every time an error occurs in my application and Ignition does it's thing a hash / anchor is added to the current url like somesite.test/**#F50**
Why does this happen? And how can I disable this 'feature'? It's very annoying if the app is using hash/anchors itself.

**Comment by nickdekruijk** [comment]
I can't imagine I'm the only one who has this nuisance?

**Reply by AlexVanderbist** [reply]
This was a bug where the stackframe number would be added to the window's URL. It's been fixed a while ago :)
